Templates/Swift/Sources/APIClient.swift

```diff
@@ -184,20 +191,26 @@ public class APIClient {
     }
 }
 
+private extension APIClient {
+    enum InternalError: Error {
```

Comment:
Happy to rename this (and its cases) to something different. It's mostly to avoid crashes at any cost.